### PR TITLE
Update TELOPS DUT Send To Cert card sourced from the CQT card

### PR DESCRIPTION
**Pull request by clairlin53**
Enhance the description details to Jira TELOPS DUT Send To Cert card from the CQT hardware transfer card. Additionally, update the reporter of the TELOPS card to match the assignee of CQT card.

https://warthogs.atlassian.net/browse/CQT-3130 

1. Copy the CQT content to TELOPS card.
2. Remove ''qa_launchpad_id''.
3. Modify: if the CID has already exist in the google_sheet, it will not raise the exception.
4. Bypass if the HIC connection is down.

### CODE REVIEW

File: Tools/PC/transfer-hw-to-cert/tests/test_cqt_handler.py [hunk 1]
```diff
@@ -129,9 +131,14 @@ def test_get_expected_result(
         mock_get_content_from_a_jira_card.return_value = \
             VALID_CONTENT_FROM_API
 
+        
         expected_result = {
             'gm_image_link': 'https://oem-share.canonical.com/partners/sutton/share/bachman/sutton-workstation-2022-10-07/pc-sutton-bachman-focal-amd64-X00-20221004-139.iso',  # noqa: E501
-            'qa_launchpad_id': 'fake-valid-launchpad_id',
+            'assignee_original_id': 'accountID',
+            'description_original_data': {
```
Comment (stanley31huang) [inline Tools/PC/transfer-hw-to-cert/tests/test_cqt_handler.py:138]: we don't need a blank line between bracket character here

File: Tools/PC/transfer-hw-to-cert/handlers/cqt_handler.py [hunk 2]
```diff
@@ -39,12 +39,16 @@ def get_content_from_a_jira_card(key: str) -> dict:
...
+    assignee_id = assignee_info.get('accountId', '')
     test_result_field = response['issues'][0]['fields'][test_result_field_id]
 
     # Return dictionary
     re_dict = {
+        'description_original_data': description_field,
+        'assignee_original_id': assignee_id,
         'gm_image_link': '',
-        'qa_launchpad_id': '',
+        #'qa_launchpad_id': '',
```
Comment (stanley31huang) [inline Tools/PC/transfer-hw-to-cert/handlers/cqt_handler.py:51]: we could just remove the qa_launchpad_id

File: Tools/PC/transfer-hw-to-cert/handlers/cqt_handler.py [hunk 3]
```diff
@@ -189,8 +172,10 @@ def get_candidate_duts(key: str) -> dict:
     # Return dictionary
     re_dict = {
         'data': [],
+        'description_original_data': content['description_original_data'],
+        'assignee_original_id': content['assignee_original_id'],
         'gm_image_link': content['gm_image_link'],
-        'qa_launchpad_id': content['qa_launchpad_id']
+        #'qa_launchpad_id': content['qa_launchpad_id']
```
Comment (stanley31huang) [inline Tools/PC/transfer-hw-to-cert/handlers/cqt_handler.py:178]: we could just remove the qa_launchpad_id

File: Tools/PC/transfer-hw-to-cert/handlers/telops_handler.py [hunk 4]
```diff
@@ -45,15 +46,18 @@ def create_send_dut_to_cert_card_in_telops(
...
-        fields['reporter']['id'] = qa_members[reporter]['jira_uid']
+        fields['reporter']['id'] = assignee_original_id
 
         # Link task back to CQT task
         update_link = telops_jira_api.create_link_issue_content(
             target_issues=[{'key': cqt_card}])
 
         issue_updates.append({'fields': fields, 'update': update_link})
-
+
```
Comment (stanley31huang) [inline Tools/PC/transfer-hw-to-cert/handlers/telops_handler.py:60]: please trim the space or tab character

File: Tools/PC/transfer-hw-to-cert/main.py [hunk 5]
```diff
@@ -78,7 +78,7 @@ def main():
             print('-' * 5 + 'Updating C3' + '-' * 5)
             update_duts_info_on_c3(
                 data=data['data'], new_holder=args.c3_holder)
-
+
```
Comment (stanley31huang) [inline Tools/PC/transfer-hw-to-cert/main.py:81]: Please trim the space or tab character

File: Tools/PC/transfer-hw-to-cert/main.py [hunk 6]
```diff
@@ -89,9 +89,11 @@ def main():
         print('-' * 5 + 'Creating card to TELOPS board' + '-' * 5)
         create_send_dut_to_cert_card_in_telops(
             cqt_card=key,
-            reporter=data['qa_launchpad_id'],
+            description_original_data=data['description_original_data'],
+            assignee_original_id=data['assignee_original_id'],
             data=data['data']
         )
+        #print(json.dumps(data['data'], indent=2))
```
Comment (stanley31huang) [inline Tools/PC/transfer-hw-to-cert/main.py:96]: we could just remove this line

File: Tools/PC/transfer-hw-to-cert/tests/test_cqt_handler.py [hunk 7]
```diff
@@ -112,7 +112,9 @@ def test_raise_exception_as_table_row_number_is_less_than_3(
         mock_get_content_from_a_jira_card.return_value = {
             'table': [1, 2],
             'gm_image_link': '',
-            'qa_launchpad_id': ''
+        #    'qa_launchpad_id': '',
```
Comment (stanley31huang) [inline Tools/PC/transfer-hw-to-cert/tests/test_cqt_handler.py:115]: please remove this line

File: Tools/PC/transfer-hw-to-cert/tests/test_cqt_handler.py [hunk 8]
```diff
@@ -129,9 +131,14 @@ def test_get_expected_result(
...
 
+        
         expected_result = {
             'gm_image_link': 'https://oem-share.canonical.com/partners/sutton/share/bachman/sutton-workstation-2022-10-07/pc-sutton-bachman-focal-amd64-X00-20221004-139.iso',  # noqa: E501
-            'qa_launchpad_id': 'fake-valid-launchpad_id',
+            'assignee_original_id': 'accountID',
+            'description_original_data': {
+
+            },
+            #'qa_launchpad_id': 'fake-valid-launchpad_id',
```
Comment (stanley31huang) [inline Tools/PC/transfer-hw-to-cert/tests/test_cqt_handler.py:141]: please remove this line

**Review by stanley31huang — APPROVED** [review]
Thanks for the changes, LGTM

**Review by baconYao — APPROVED** [review]
LGTM

**Pull request merged**